README.md

```diff
@@ -51,65 +50,78 @@ spec:
   nodeName: node02
   reason: "Test node maintenance"
 
-$ kubectl apply -f config/samples/nodemaintenance_v1beta1_nodemaintenance.yaml
+$ kubectl create -f config/samples/nodemaintenance_v1beta1_nodemaintenance.yaml
```

Comment:
Why this change?
Create works once only, apply multiple times e.g. when the file is updated

Reply:
In the context of the documentation, I have felt the verb "create" is more suitable than "apply" considering that they work the same (equivalent). I was not aware of this difference, and I agree with your point, thus I will return to "apply".